en/dronecan/ark_flow.md

```diff
@@ -72,15 +72,15 @@ The Ark Flow will not boot if there is no SD card in the flight controller when
 
 ### Enable DroneCAN
 
-In order to use the ARK Flow board, connect it to the Pixhawk CAN bus and enable the UAVCAN driver by setting parameter [UAVCAN_ENABLE](../advanced_config/parameter_reference.md#UAVCAN_ENABLE) to `2` for dynamic node allocation (or `3` if using [DroneCAN ESCs](../dronecan/escs.md)).
 
 The steps are:
 
-- In _QGroundControl_ set the parameter [UAVCAN_ENABLE](../advanced_config/parameter_reference.md#UAVCAN_ENABLE) to `2` or `3` and reboot (see [Finding/Updating Parameters](../advanced_config/parameters.md)).
-- Connect ARK Flow CAN to the Pixhawk CAN.
+- In _QGroundControl_ set the parameter [UAVCAN_ENABLE](../advanced_config/parameter_reference.md#UAVCAN_ENABLE) to `2` for dynamic node allocation (or `3` if using [DroneCAN ESCs](../dronecan/escs.md)) and reboot (see [Finding/Updating Parameters](../advanced_config/parameters.md)).
+- Connect ARK Flow MR CAN to the Pixhawk CAN.
 
 Once enabled, the module will be detected on boot.
-Flow data should arrive at 10Hz.
+Flow data should arrive at 100Hz.
+Distance sensor data should arrive at 40Hz
 
 DroneCAN configuration in PX4 is explained in more detail in [DroneCAN > Enabling DroneCAN](../dronecan/index.md#enabling-dronecan).
 
```

Comment:
Below, is good, but should we point to the setup in https://docs.px4.io/main/en/dronecan/#optical-flow-sensor instead/as well?